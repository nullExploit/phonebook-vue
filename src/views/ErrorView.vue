<template>
    <h1 style="text-align: center; margin-top: 2rem;">Error 404</h1>
    <p style="text-align: center;">not found</p>
</template>