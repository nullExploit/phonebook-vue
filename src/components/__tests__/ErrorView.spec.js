import ErrorView from "@/views/ErrorView.vue";
import { describe, expect, it } from "vitest";
import { mount } from "@vue/test-utils";


describe("Error page", () => {
  it("Should return error page", () => {
    const data = mount(ErrorView);

    expect(data.html()).toMatchSnapshot()
  });
});
