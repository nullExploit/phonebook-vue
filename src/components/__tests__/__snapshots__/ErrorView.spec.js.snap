// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error page > Should return error page 1`] = `
"<h1 style="text-align: center; margin-top: 2rem;">Error 404</h1>
<p style="text-align: center;">not found</p>"
`;
